test/compare.bash

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+function line() {
+    echo ================================================================================
+}
+
+if [ $# != 4 ]; then
+    echo "Usage: compare.bash <target> <one test> <Aeneas before> <Aeneas after>"
+    exit 1
+fi
+
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+target=$1
+shift
+
+TEST=$1
```

Comment:
nit: I wouldn't normally shift off arguments unless I was planning to use $@ later.